### PR TITLE
🚀 Remove 'use strict' directives in post compilation transform for modu…

### PR DESCRIPTION
**Pull request by kristoferbaxter**
When loading the module runtime, documents indicate the module loader system should be used via a `type=module` attribute on corresponding `HTMLScriptElement`s.

This PR removes 'use strict' directives from the module output, since [modules use strict mode by default](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#Strict_mode_for_modules).

### CODE REVIEW

**Comment by rsimha** [comment]
Nice work, @kristoferbaxter. Is it desirable to clean up all the `use strict` directives in our code in addition to doing so during post-closure transforms?

**Comment by kristoferbaxter** [comment]
@rsimha We can remove them safely from the module output, but not in our code until we move over to using EcmaScript Modules for all our node tooling code and depdencies.

TLDR: Not yet.
